### PR TITLE
Update week_one_check-Robbie Smith.md

### PR DESCRIPTION
**Pull request by Robbie-Smith**

### CODE REVIEW

File: module-2/backend/week_one.md [hunk 1]
```diff
@@ -5,30 +5,78 @@ Fork this respository. Answer the questions to the best of your ability. Try to
...
+  * POST
+  * PUT
+  * DELETE
+  * PATCH
 2. What is Sinatra?
+  * It's a framework for building websites. A domain specific language
 4. What is MVC?
+  * It's a convention for writing web applications. Model, View, Controller
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  * So that other people can understand what we are doing.
```
Comment (s-espinosa) [inline module-2/backend/week_one.md]: 👍

File: module-2/backend/week_one.md [hunk 2]
```diff
@@ -5,30 +5,78 @@ Fork this respository. Answer the questions to the best of your ability. Try to
...
+    @count = 1
     erb :index
   end
+  
+  index.erb
+  <p><%= @count %></p>
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
+  erb :index, :locals => {:number => @@number,
```
Comment (s-espinosa) [inline module-2/backend/week_one.md]: Don't think we need this piece. Maybe you copied in and forgot to delete?

File: module-2/backend/week_one.md [hunk 3]
```diff
@@ -5,30 +5,78 @@ Fork this respository. Answer the questions to the best of your ability. Try to
...
 10. Why do I need a development AND test database?
+  * Our testing would mess up the development database by adding, updating, and deleting. We also need to have a fresh empty database each time we test. This is why we have the test database so we can test and keep out testing separate from the development.
 11. What's responsive design?
+  * It is when we design a website that can be responsive to different screens(e.g. mobile to desktop and vice versa) and still be usable.
 12. What is CRUD and why is it important?
+  * Create, Read, Update, and Delete. It allows a user to interact with a database giving them full access to the data.
 13. What does HTTP stand for? 
+  * Hyper text transfer protocol
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+  * <%= %> (this displays ruby on the website), <% %> this allows the ruby to run without being displayed.
```
Comment (s-espinosa) [inline module-2/backend/week_one.md]: 👍

File: module-2/backend/week_one.md [hunk 4]
```diff
@@ -5,30 +5,78 @@ Fork this respository. Answer the questions to the best of your ability. Try to
...
 15. What's an ORM?
+  * Object Relation Model. It allows to relate objects with a database.
 16. What's the most commonly used ORM?
+  * SQL
```
Comment (s-espinosa) [inline module-2/backend/week_one.md]: SQL is a database language. ActiveRecord is the ORM we're using (and most commonly used)

File: module-2/backend/week_one.md [hunk 5]
```diff
@@ -5,30 +5,78 @@ Fork this respository. Answer the questions to the best of your ability. Try to
...
+  * '/restraunts/new
+    *Adds a new restaurant
+  * '/restaurants/:id/edit'
+    *Shows the user the form that pdates/edits the current information for the current restaurant
+  * '/restaurants/:id'
+    *Brings the user to the newly updated restaurant
+  * '/restaurants/:id'
+    *Allows the user to delete the restaurant
+  * '/restaurants'
+    *After the deletion, the user is redirected to the page that shows all restaurants.
```
Comment (s-espinosa) [inline module-2/backend/week_one.md]: These URL patterns need to be paired with verbs to really get the whole picture. So, for example, you have `/restaurants/:id` a few times above, but that pattern does different things depending on the verb. So a GET to that path will show a specific restaurant, while a PUT will update that restaurant.

File: module-2/backend/week_one.md [hunk 6]
```diff
@@ -5,30 +5,78 @@ Fork this respository. Answer the questions to the best of your ability. Try to
...
+  
 18. What's a migration? 
+  * A migration is when you add a column/row to the database and update the schema
 19. When you create a migration, does it automatically modify your database?
+  * No it allows you to write the code that will modify the database
```
Comment (s-espinosa) [inline module-2/backend/week_one.md]: 👍

File: module-2/backend/week_one.md (hunk 5 above)
Comment (Robbie-Smith) [inline module-2/backend/week_one.md]: Thanks!

**Pull request closed without merging**